tests/test_tasks/test_all_abstasks.py

```diff
@@ -31,6 +30,17 @@
 tasks = [t for t in MTEB().tasks_cls if t.metadata.name not in ALL_MOCK_TASKS]
 
 
+dataset_revisions = list(
+    {  # deduplicate as multiple tasks rely on the same dataset (save us at least 100 test cases)
+        (t.metadata.dataset["path"], t.metadata.dataset["revision"])
+        for t in mteb.get_tasks(exclude_superseded=False)
+        if not isinstance(t, (AbsTaskAggregate, AbsTaskSpeedTask))
+        and t.metadata.name != "AfriSentiLangClassification"
```

Comment:
Can we add this back?
```suggestion
        and t.metadata.name != "AfriSentiLangClassification" # HOTFIX: Issue#1777. Remove this line when issue is resolved.
```

Reply:
Still 404. Should we delete the dataset?